src/components/CategoryFilter.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ handleFilterChange }) => {
+  const Category = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
```

Comment:
- [ ] You are using this array in the `BooksForm` component, right? Why don't you put this array in a `utils` folder and export it? Then you could import it here and in the form to make your code cleaner. If you ever need to change categories, you will only need to change them in one file. ALso, remember that variables in js are written in lowercase if they are not components, or UPPERCASE if they are constants.